### PR TITLE
Change content of dotnet artifact outputs to facilitate correct coverage report creation

### PR DESCRIPTION
**Pull request by djorgensendk**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -78,20 +78,20 @@ jobs:
   dotnet_ci_build:
     needs: changes
     if: ${{ needs.changes.outputs.dotnet == 'true' }}
-    uses: Energinet-DataHub/.github/.github/workflows/dotnet-build-prerelease.yml@v8
+    uses: Energinet-DataHub/.github/.github/workflows/dotnet-build-prerelease.yml@poc-test-coverage
```
Comment (djorgensendk) [inline .github/workflows/ci.yml:81]: Must be reverted when https://github.com/Energinet-DataHub/.github/pull/162 has been merged

File: .github/actions/dotnet-tests-prepare-outputs/action.yml [hunk 2]
```diff
@@ -54,7 +54,9 @@ runs:
     - name: Copy all bin folders
```
Comment (dstenroejl) [inline .github/actions/dotnet-tests-prepare-outputs/action.yml:54]: This name is not correct anymore

Comment (dstenroejl) [reply .github/actions/dotnet-tests-prepare-outputs/action.yml:54]: If we always need all files (except obj), then I guess we do not have to copy everything to another folder. The workflow which calls this action could just use the "source/dotnet" folder. Let me explain when we talk. :)

**Review by dstenroejl — COMMENTED** [review]
Commented

**Review by dstenroejl — APPROVED** [review]

**Pull request merged**